src/components/sections/employees/employees.module.css

```diff
@@ -61,3 +60,32 @@
   flex-wrap: wrap;
   gap: 1rem;
 }
+
```

Comment:
These changes should probably be in EmployeeCard, not EmployeeList

Reply:
I'm curious what i've done here 😅 😆